src/redux/books/books.js

```diff
@@ -1,28 +1,50 @@
 const REMOVE = 'REMOVE';
-const ADD = 'ADD';
+const DISPLAY = 'DISPLAY'
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/W1RT9D7Mwtti9RBvC75J/books'
 
+const displayData = (book) => ({
+  type: DISPLAY,
+  book
+})
 
-const books = [];
-
-// remove books
+// remove books action
 export const removeBook = (id) => ({
   type: REMOVE,
   id,
 });
 
+// fetch books 
+export const fetchBooks = () => (dispatch) => {
+  fetch(url)
+  .then(res => res.json())
+  .then(data => dispatch(displayData(data)))
+}
```

Comment:
- [ ] [OPTIONAL] Kindly use async and await for the `fetchBooks` function as it makes your code cleaner.